### DISCUSSION TITLE
Is there a way to boot an mdraid raid 1 ext4 volume directly from the UFI image?

### DISCUSSION THREAD

**Discussion by lol-catz** [zbm-dev/zfsbootmenu#605] [General]
Some of my machines run a zfs root mirror, and some need to run a ext4 mdraid mirror for performance reasons..
Is there a way to boot an mdraid raid 1 ext4 volume directly from the UFI image?

**Comment by classabbyamp** [comment]
zfsbootmenu, as the name suggests, only supports booting ZFS-on-root environments. 

If you want to boot something else and a ZFS-on-root environment on the same machine, I would suggest using another bootloader (like rEFInd) to launch zfsbootmenu.
